content/millennium/r4/proprietary-codes.md

```diff
@@ -360,6 +404,17 @@ This code set describes why an immunization was not administered when documented
     { 
       "system": "https://fhir.cerner.com/<EHR source id>/codeSet/30440", 
       "code": "688466", 
-      "display": "Postpone due to refusal", 
+      "display": "Postpone due to refusal",
+      "userSelected": true
+    } 
+
+#### Code Set 4002640 Correspondence Status
+
+This code set includes Functional Statement Account Statuses such as Pending and Delivered.
```

Comment:
Kind of confusing the code set is called Correspondence Status but it includes Functional Statement Account Statuses?

Reply:
Talked with Alex about this, we determined to do:

This code set includes the statuses that reflect the current status of a correspondence within the statement workflow such as Pending and Delivered.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/291/commits/a7abfb39bc998b699f7ed95a027e4faa8eb75a9b#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R413

Reply:
+1